packages/hydrogen-demo-site/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
NIT: There is a lot of dot files added in this demo that are not used in other demo files - do we actually want them? Some of `.eslintignore` or `.gitignore` are already covered by rules in root ignore files for example. Not necessarily hurting anything, I just find it confusing to have multiple of those dot files sprinkled in nested directories

Reply:
Ah I just realized most of it was copied from https://github.com/Shopify/hydrogen/tree/main/templates/demo-store - are there any change in the site other than addition of `netlify.toml` and adjustments to `remix.config.js` and minor `README.md` to show how to use netlify-cli?

Reply:
To expand on above question - for future maintenance it would probably be good to list in README of site that it's demo site copied directly from existing hydrogen template and list changes made to it (if there was anything that I didn't see/think of checking on what's the difference). Otherwise it might be a pain if someone else will ever have to update that demo